recipes/bioconductor-rhdf5lib/configure.patch

```diff
@@ -0,0 +1,11 @@
+--- configure	2018-12-20 17:09:38.001376321 +0100
++++ configure.patched	2018-12-20 17:25:42.834031679 +0100
```

Comment:
You probably want `configure` rather than `configure.patched` there.

Reply:
Isn't that only cosmetic?